src/schema/schema_create.c

```diff
@@ -212,7 +218,7 @@ __create_file(
         if (!import_repair) {
             WT_ERR(__wt_scr_alloc(session, 0, &val));
             WT_ERR(__wt_buf_fmt(session, val,
-              "id=%" PRIu32 ",version=(major=%d,minor=%d),checkpoint_backup_info=,checkpoint_lsn=",
+              "id=%" PRIu32 ",version=(major=%d,minor=%d),checkpoint_lsn=",
```

Comment:
Add a comment here about where `checkpoint_backup_info=` gets initialized.

Reply:
Added a comment, let me know if you like the comment or needed anymore. Since I have added a comment above when we add the **checkpoint_backup_info=** information.